code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -52,6 +52,7 @@
 	var/skin_tone = ""
 	var/body_gender = ""
 	var/species_id = ""
+	var/is_husked = FALSE
```

Comment:
Why add another variable for this when we have `TRAIT_HUSK`?

Reply:
reason for this was TG parity, in their current implementation, body parts have this var/is_husked and most husk rendering logic works off that